drivers/watchdog/wdt_mcux_imx_wdog.c

```diff
@@ -0,0 +1,167 @@
+/*
+ *
+ * Copyright (c) 2018, NXP
+
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_imx_wdog
+
+#include <drivers/watchdog.h>
+#include <fsl_wdog.h>
+
+#define LOG_LEVEL CONFIG_WDT_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(wdt_mcux_wdog);
+
+#define WDOG_TMOUT_SEC(x)  (x * 2 / 1000 - 1)
```

Comment:
I think this is taking the configured window size in milliseconds and returning the floor of half the duration, in seconds.

Using `MSEC_PER_SEC` instead of 1000 would be better if that's correct.

Reply:
> I think this is taking the configured window size in milliseconds and returning the floor of half the duration, in seconds.
> 
> Using `MSEC_PER_SEC` instead of 1000 would be better if that's correct.

Hi，forget an (int) to cast the value. Has Updated with (int) and also the MSEC_PER_SEC